_data/fr/general.yml

```diff
@@ -0,0 +1,13 @@
+footer:
+  copyright: Logiciel développé par <a href="https://github.com/corrados" target="_blank" rel="noreferrer">Volker Fischer</a> et les <a href="https://github.com/corrados/jamulus/graphs/contributors" target="_blank" rel="noreferrer">contributeur·trice·s</a>
```

Comment:
"contributeur·trice·s" : it isn't proper French. That means this kind of writing is pushing all the people not politically aligned with this. That roughly means leaving out ~99% of the Jamulus potential French readers users. I'm leaving this message here, but that's all over this draft translation.

Reply:
C'est un choix délibéré, sans être un militant féministe un choix politique aussi, la langue française en une langue vivante, tant sur le plan oral qu'écrit. Si trouver une solution (avec l'écriture dite inclusive en utilisant le point médian) pour dé-masculiniser un peu notre langue, pourquoi pas. Je n'ai pour ma part aucun problème avec ça. Et si j'y étais retissant, j'ai revu ma position. 
Lorsque le contexte s'y prête j'ai formulé autrement que par l'utilisation de l'écriture inclusive.

/cc @Saimyx

Reply:
Moi j'y suis opposé. Car écrire comme ça, est un choix politique, voire idéologique. Le but est de faire de la traduction, pas de rendre la lecture du français plus compliqué qu'il ne l'est déjà.